sysutils/mackup/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                mackup
-version             0.8.36
+version             0.8.37
 
 categories-prepend  sysutils
 license             GPL-3
```

Comment:
this should be `GPL-3+; see [here](https://github.com/lra/mackup/blob/3d2d41c3763820e3949c741536c1ced78fdcc214/setup.py#L25C1-L25C62) and in the license file on upstream's GitHub.